cypress/integration/calculator.spec.js

```diff
@@ -0,0 +1,111 @@
+            .get('#integerSelect').should('not.be.selected')
+            .get('#errorMsgField').should('not.be.visible')
+    })
+
+    it('Check if fourth build with operation "Subtract" works properly', () => {
+        cy.get('#selectBuild').select('4')
+            .get('#number1Field').type(4)
+            .get('#number2Field').type(2.5)
+            .get('#selectOperationDropdown').select('1')
+            .get('#calculateButton').click()
+            .get('#numberAnswerField').should('have.value', 1.5)
+            .get('#integerSelect').click()
+            .get('#numberAnswerField').should('have.value', 1)
+            .get('#clearButton').click()
+            .get('#numberAnswerField').should('have.value', '')
+            .get('#integerSelect').should('not.be.selected')
+            .get('#errorMsgField').should('not.be.visible')
+    })
+
+
+    it('Check if sixth build with operation "Multiply" works properly', () => {
+        cy.get('#selectBuild').select('6')
+            .get('#number1Field').type(5)
+            .get('#number2Field').type(2.5)
+            .get('#selectOperationDropdown').select('2')
+            .get('#calculateButton').click()
+            .get('#numberAnswerField').should('have.value', 12.5)
+            .get('#integerSelect').click()
+            .get('#numberAnswerField').should('have.value', 12)
+            .get('#clearButton').click()
+            .get('#numberAnswerField').should('have.value', '')
+            .get('#integerSelect').should('not.be.selected')
+            .get('#errorMsgField').should('not.be.visible')
+    })
+
+  
+
+    it('Check if ninth build with operation "Divide" works properly', () => {
+        cy.get('#selectBuild').select('9')
+            .get('#number1Field').type(3)
+            .get('#number2Field').type(2)
+            .get('#selectOperationDropdown').select('3')
+            .get('#calculateButton').click()
+            .get('#numberAnswerField').should('have.value', 1.5)
+            .get('#integerSelect').click()
+            .get('#numberAnswerField').should('have.value', 1)
+            .get('#clearButton').click()
+            .get('#numberAnswerField').should('have.value', '')
+            .get('#integerSelect').should('not.be.selected')
+            .get('#errorMsgField').should('not.be.visible')
+})    
+
+
+
+const firstInputData = 6
+const secondInputData = 1.5
+
+const builds = ['1', '2', '3', '4', '5', '6', '7', '8', '9']
+
+const operations = ['Add', 'Subtract', 'Multiply', 'Divide', 'Concatenate']
+
+const calculationResults = { Add: 7.5, Subtract: 4.5, Multiply: 9, Divide: 4, Concatenate: '61.5' }
+const integerOnlyCalculationResults = { Add: 7, Subtract: 4, Multiply: 9, Divide: 4 }
+
+describe('Tests with loop', () => {
+    beforeEach("Executes before each test",() => {
+        cy.visit("/BasicCalculator")
+    })
+
+    for (let i = 0; i < builds.length; i++) {
+        for (let x = 0; x < operations.length; x++) {
```

Comment:
It is great to use parameterization but loop in a loop might be too much. Such tests loose their readability. I would suggest you to use the loop only for different _builds_. 
Yes, this means you need to write more tests, but:
* you may rethink which tests are needed and which not
* you may use different flows for different operations (and different inputs! sometimes it is needed, especially for _division_)
* you will rid of 'if' statement (line 98-100)
Also, rethink variable names in that loop. It would be more readable to have 'build' instead of 'i' and 'operation' instead of 'x'. 
![meaningful-variable-names-or-nah-69244383](https://user-images.githubusercontent.com/33998882/92249622-0d7c9500-eed3-11ea-9231-65c6628a1568.png)